### PR TITLE
Restructure logic to list out bookmarks using `bookmark list`.

### PR DESCRIPTION
**Pull request by josephhhhhhhhh**
For #244 .

### CODE REVIEW

File: src/main/java/seedu/eduke8/exception/ExceptionMessages.java [hunk 1]
```diff
@@ -41,4 +41,6 @@
     public static final String ERROR_BOOKMARK_DELETE_IOB_ERROR = "Please choose an appropriate index number to be "
             + "deleted from the list!";
     public static final String ERROR_BOOKMARK_NONE = "You currently do not have any bookmarks stored.";
+    public static final String ERROR_BOOKMARK_INCORRECT_COMMAND = "Invalid command! The command for bookmark in the "
+            + "main menu is as follows:\n'bookmark list' or 'bookmark delete'";
```
Comment (Zhi-You) [inline src/main/java/seedu/eduke8/exception/ExceptionMessages.java:45]: Should this be a System.lineSeparator() instead?

File: src/main/java/seedu/eduke8/parser/MenuParser.java [hunk 2]
```diff
@@ -101,7 +106,7 @@ public Command parseCommand(DisplayableList topicList, String userInput) {
             int numOfQuestions = 0;
             String topicName = "";
             int userTimer = 0;
-            if ((commandArr.length > 4 && (!commandArr[3].contains("/") || commandArr[4].equals("")))
+            if ((commandArr.length > 4 && (!commandArr[3].contains("/") || !commandArr[4].equals("")))
```
Comment (Zhi-You) [inline src/main/java/seedu/eduke8/parser/MenuParser.java:109]: Great change to capture edge cases.

File: src/main/java/seedu/eduke8/parser/MenuParser.java [hunk 3]
```diff
@@ -154,11 +158,15 @@ public Command parseCommand(DisplayableList topicList, String userInput) {
...
-                return new IncorrectCommand(e.getMessage());
+            } else if (commandArr.length >= BOOKMARK_LIST_COMMAND_ARR_SIZE
+                    && commandArr[1].trim().equalsIgnoreCase(COMMAND_BOOKMARK_LIST)) {
+                try {
+                    return new BookmarkCommand(BOOKMARK_LIST, bookmarkList);
+                } catch (Eduke8Exception e) {
+                    return new IncorrectCommand(e.getMessage());
+                }
+            } else {
+                return new IncorrectCommand(ERROR_BOOKMARK_INCORRECT_COMMAND);
```
Comment (Zhi-You) [inline src/main/java/seedu/eduke8/parser/MenuParser.java:169]: Looks good!

**Review by Zhi-You — APPROVED** [review]
LGTM!

**Comment by kstonekuan** [comment]
Closes #244

**Pull request merged**